wallet/connect/index.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Connect to MetaMask using various options.
+---
+
+# Connect to MetaMask
+
+You can connect your dapp to MetaMask using one of the following methods:
+
+- [MetaMask SDK](metamask-sdk/index.md)
+- Third-party libraries with SDK support, such as [Wagmi](3rd-party-libraries/wagmi.md) or
+  [Web3-Onboard](3rd-party-libraries/web3-onboard.md)
+- The [Wallet API](wallet-api.md) directly
+
+The following table compares the supported features of each connection option:
+
+| Feature                                                                                                                                                                                                                                                                                            | MetaMask SDK | Third-party libraries with SDK | Wallet API only |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------:|:------------------------------:|:---------------:|
+| Connect from a web dapp to the MetaMask extension                                                                                                                                                                                                                                                  |      ✓       |               ✓                |        ✓        |
+| Connect from a web dapp, desktop, mobile, and gaming dapps to MetaMask Mobile                                                                                                                                                                                                                      |      ✓       |               ✓                |                 |
+| Connect to MetaMask using the [EIP-6963 provider](../concepts/wallet-interoperability.md)                                                                                                                                                                                                          |      ✓       |               ✓                |        ✓        |
```

Comment:
It's a little complex because the EIP-6963 mechanism is meant to support multi-wallet detection. However, the SDK only uses it to detect MetaMask without conflicting with other wallets, but is not a multi-wallet detector. I've updated this to "Use EIP-6963 to connect to MetaMask without conflict."